src/main/java/com/iota/iri/conf/DbConfig.java

```diff
@@ -6,32 +6,44 @@
 public interface DbConfig extends Config {
 
     /**
-     * @return Descriptions#DB_PATH
+     * Default Value: {@value BaseIotaConfig.Defaults#DB_PATH}
+     *
+     * @return {@value DbConfig.Descriptions#DB_PATH}
      */
     String getDbPath();
 
     /**
-     * @return {@value Descriptions#DB_LOG_PATH}
+     * Default Value: {@value BaseIotaConfig.Defaults#DB_LOG_PATH}
+     *
+     * @return {@value DbConfig.Descriptions#DB_LOG_PATH}
      */
     String getDbLogPath();
 
     /**
-     * @return {@value Descriptions#DB_CACHE_SIZE}
+     * Default Value: {@value BaseIotaConfig.Defaults#DB_CACHE_SIZE}
+     *
+     * @return {@value DbConfig.Descriptions#DB_CACHE_SIZE}
      */
     int getDbCacheSize();
 
     /**
-     * @return {@value Descriptions#MAIN_DB}
+     * Default Value: {@value BaseIotaConfig.Defaults#ROCKS_DB}
```

Comment:
the default value name does not match the description name.
`ROCKS_DB` vs `MAIN_DB`